cmd/reconciler-manager/main.go

```diff
@@ -70,10 +70,12 @@ func init() {
 func main() {
 	var metricsAddr string
 	var enableLeaderElection bool
+	var allowVerticalScale bool
 	flag.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
 	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
+	flag.BoolVar(&allowVerticalScale, "allow-vertical-scale", false, "Allowing vertical scale to let VPA manage the resource")
```

Comment:
Can you define an environment variable and load the default value from the env variable? This provides more flexibility for OSS users to set the flag.

Reply:
As per other comment, this has been updated to use EnvBool